sdk/servicebus/service-bus/src/serviceBusClient.ts

```diff
@@ -438,6 +438,19 @@ export class ServiceBusClient {
       options3
     );
 
+    const operationTimeout =
+      this._clientOptions?.retryOptions?.timeoutInMs ?? Constants.defaultOperationTimeoutInMs;
+    // The number of milliseconds to use as the basis for calculating a random jitter amount
+    // opening receiver links. This is intended to ensure that multiple
+    // session operations don't timeout at the same exact moment.
+    const jitterBaseInMs = Math.min(operationTimeout * 0.01, 100);
+    // Subtract a random amount up to 100ms from the operation timeout as the jitter when attempting to open next available session link.
+    // This prevents excessive resource usage when using high amounts of concurrency and accepting the next available session.
+    // Take the min of 1% of the total timeout and the base jitter amount so that we don't end up subtracting more than 1% of the total timeout.
+    const timeoutWithJitterInMs = operationTimeout - jitterBaseInMs * Math.random();
```

Comment:
I made an update to the .NET implementation to subtract an additional 20ms as long as the timeout is over 1 s. https://github.com/Azure/azure-sdk-for-net/pull/31069

Reply:
I port this over too and add some fix so this really only applies to acceptNextSession()